src/TTextEdit.cpp

```diff
@@ -1151,9 +1188,13 @@ void TTextEdit::mousePressEvent( QMouseEvent * event )
     }
     if( event->button() == Qt::LeftButton )
     {
+        if( QGuiApplication::queryKeyboardModifiers().testFlag(Qt::ControlModifier) ) {
+          mCtrlSelecting = true;
+        }
         int x = event->x() / mFontWidth;
         if( mShowTimeStamps )
         {
+            if( x < 13 ) mCtrlSelecting = true;
```

Comment:
_Humm_, a single line `if(`...`) single statement;`

All together now:
> ..."brace with `{`...`}` even single line `if(`...`) {`code statement(s)`;}` and put the code block on the next line"... 

I'm not trying to be an awkward code style pit-nicker, just that it is far too easy (for decrepit coders like me!)  to misread the following line (the `x -= 13;`) as being what is controlled by the `if`. :stuck_out_tongue_winking_eye: